static/app/components/feedback/feedbackItem/tagsSection.tsx

```diff
@@ -1,36 +1,52 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Collapsible from 'sentry/components/collapsible';
 import {KeyValueTable, KeyValueTableRow} from 'sentry/components/keyValueTable';
 import TextOverflow from 'sentry/components/textOverflow';
 import {Tooltip} from 'sentry/components/tooltip';
+import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 
 interface Props {
   tags: Record<string, string>;
-  collapsed?: boolean;
 }
 
-export default function TagsSection({tags, collapsed}: Props) {
+export default function TagsSection({tags}: Props) {
   const entries = Object.entries(tags);
 
   return (
-    <KeyValueTable
-      noMargin
-      style={
-        collapsed
-          ? {maxHeight: '90px', overflow: 'hidden', marginBottom: `${space(1)}`}
-          : undefined
-      }
-    >
-      {entries.map(([key, value]) => (
-        <KeyValueTableRow
-          key={key}
-          keyName={key}
-          value={
-            <Tooltip showOnlyOnOverflow title={value}>
-              <TextOverflow>{value}</TextOverflow>
-            </Tooltip>
-          }
-        />
-      ))}
+    <KeyValueTable noMargin>
+      <Collapsible
+        maxVisibleItems={3}
+        collapseButton={({onCollapse}) => (
+          <StyledButton priority="primary" size="xs" onClick={onCollapse}>
+            {t('Collapse')}
+          </StyledButton>
+        )}
+        expandButton={({onExpand}) => (
+          <StyledButton priority="primary" size="xs" onClick={onExpand}>
+            {t('Expand')}
```

Comment:
What about...

"See all tags" and "See less tags" to give the button even more context?